src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -420,4 +421,58 @@ public void setNotFailingStatuses_SkipsNullValues() {
         // then
         assertThat(configuration.getNotFailingStatuses()).containsExactly(notFailingStatus);
     }
+
+    @Test
+    public void getOutputHandlers_shouldContainFilesystemOutputHandler() {
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        assertThat(configuration.getOutputHandlers()).hasSize(1);
+        assertThat(configuration.getOutputHandlers().get(0)).isInstanceOf(FilesystemOutputHandler.class);
+    }
+
+    @Test
+    public void clearOutputHandlers_shouldRemoveAllOutputHandlersFromOutputHandlersList() {
+        //given
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        assertThat(configuration.getOutputHandlers()).hasSize(1);
+        assertThat(configuration.getOutputHandlers().get(0)).isInstanceOf(FilesystemOutputHandler.class);
+
+        //when
+        configuration.clearOutputHandlers();
+
+        //then
+        assertThat(configuration.getOutputHandlers()).isEmpty();
+    }
+
+    @Test
+    public void addOutputHandler_shouldAddGivenHandlerToListOfOutputHandlers() {
+        //given
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        configuration.clearOutputHandlers();
+        assertThat(configuration.getOutputHandlers()).isEmpty();
+
+        //when
+        configuration.addOutputHandler(new FilesystemOutputHandler());
+
+        //then
+        assertThat(configuration.getOutputHandlers()).hasSize(1);
+        assertThat(configuration.getOutputHandlers().get(0)).isInstanceOf(FilesystemOutputHandler.class);
```

Comment:
you don't need to validate only class, you can validate exact reference which you have added before

Reply:
updated assert